lib/broadside/target.rb

```diff
@@ -93,5 +93,16 @@ def to_h
         Cluster: @cluster
       }
     end
+
+    def update_safe_service_config
```

Comment:
maybe this could just be called `update_service_config` or `service_config_for_updates`?  "safe" doesn't have much explanatory context in this power.